magda-web-client/src/Components/Dataset/Add/DatasetAddCommon.ts

```diff
@@ -227,6 +229,15 @@ type Access = {
     notes?: string;
 };
 
+function getInternalDatasetSourceAspectData() {
+    return {
+        id: "magda",
```

Comment:
We probably need to allow slightly better for when a instance of Magda is crawling another instance of Magda - like maybe this should be "magda-internal" or something? And the name "_This_ Magda metadata creation tool"? 🤔

Reply:
@AlexGilleran 
Yeah~ I think it's good idea to provide more details on this to make it more future proof.
Do we want to use the `type` field to provide more details on the source?
Otherwise, it looks like:
id: `magda-internal`, type: `internal`
Probably, id: `magda`, type: `authoring tool`?

Reply:
Maybe we pass in an id for the instance like `magda-data-gov-au` or `magda-dawe`, then have `type: internal` still? 🤔 .

I think ideally when another magda ingests this, it should know to copy the `id` but change the `type` to `external` or something.

Reply:
If we go for the `magda-data-gov-au` or `magda-dawe` for the `id`, we probably need an extra config option?
For a similar purpose (i.e. able to tell what's the original Magda system create it), will the current `url` field (of source aspect) be a good alternative without adding an extra config option? I mean we probably can set it to `global.externalUrl`?

Reply:
Yeah using `global.externalUrl` sounds like a good idea!